opium/test/cookie.ml

```diff
@@ -0,0 +1,32 @@
+let headers = [ "Cookie", "yummy_cookie=choco; tasty_cookie=strawberry" ]
```

Comment:
This is not going to be run I think, you need to add the module to the dune file in `opium/test`

Reply:
Thanks, fixed.